YeogiEottae/YeogiEottae/Presentation/Common/UIComponents/YeogiToast/YeogiToast.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  YeogiToast.swift
+//  YeogiEottae
+//
+//  Created by Seonwoo Kim on 5/18/24.
+//
+
+import UIKit
+
+import SnapKit
+
+@frozen
+enum ToastType {
+    case soldout
+    case deinitLike
+    case addHotelLike
+    case addRoomLike
+    case addCompare
+}
+
+final class YeogiToast {
+    static func show (type: ToastType, duration: TimeInterval = 1, isTabBar: Bool = true, completion: (() -> Void)? = nil) {
+        let toastView = YeogiToastView(type: type)
+        let toastWidth:Int
+        let toastHeight:Int
+        
+        switch type {
+        case .soldout:
+            toastWidth = 184
+            toastHeight = 42
+        case .deinitLike:
+            toastWidth = 235
+            toastHeight = 44
+        case .addHotelLike:
+            toastWidth = 314
+            toastHeight = 60
+        case .addCompare:
+            toastWidth = 336
+            toastHeight = 48
+        case .addRoomLike:
+            toastWidth = 297
+            toastHeight = 48
+        }
+        
+        guard let window = UIWindow.current else { return }
+        window.subviews
+            .filter { $0 is YeogiToast }
```

Comment:
```suggestion
            .filter { $0 is YeogiToastView }
```
와 같이 바꾸는 게 맞는 것 같습니다!~